backend/resolver/aws/aws.go

```diff
@@ -164,6 +165,13 @@ func (r *res) Resolve(ctx context.Context, wantTypeURL string, input proto.Messa
 func (r *res) Search(ctx context.Context, typeURL, query string, limit uint32) (*resolver.Results, error) {
 	switch typeURL {
 	case typeURLInstance:
+		// Output from kubectl when getting nodes has them in the format of: aws:///us-east-1/i-0f8f8f8f
+		// To allow users to use this format in queries, we can remove the prefix.
+		// Note that if we don't do this, the aws:// will be considered the account due to the pattern matching
+		// done via the protobuf.
+		// TrimPrefix returns s without the provided leading prefix string. If s doesn't start with prefix, s is returned unchanged.
+		query = strings.TrimPrefix(query, "aws:///")
```

Comment:
I dont think hard coding here is the way to go just for kubectl output, we should think of a different way, maybe something thats configurable. Or just got back to the drawing board rethink this whole issue.